src/System.IO/src/System.IO.csproj

```diff
@@ -17,6 +17,13 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
   </PropertyGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\System.Diagnostics.Debug\src\System.Diagnostics.Debug.csproj">
```

Comment:
I modeled it after the [System.Collections Debug inclusion](https://github.com/dotnet/corefx/blob/master/src/System.Collections/src/System.Collections.csproj). Adding a package reference in the project.json blew everything up e.g. "System.Object not defined"